<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ведомость посещаемости</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Группа</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_group">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Неделя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_week">
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24.10 - 28.10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>31.10 - 04.11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07.11 - 11.11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14.11 -18.11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21.11 -25.11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28.11 -02.12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05.12 -09.12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12.12 -16.12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_reset">
        <property name="text">
         <string>Сбросить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
